src/app/shared/classes/releases.ts

```diff
@@ -57,15 +57,18 @@ export const releases: ReleaseItem[] = [{
 }, {
   version: 'EnergyPlus 23.2.0',
   url: 'https://github.com/NREL/EnergyPlus/releases/tag/v23.2.0'  
+}, {
+  version: 'EnergyPlus 24.1.0',
+  url: 'https://github.com/NREL/EnergyPlus/releases/tag/v24.1.0'  
+}, {
 }];
 
 export const latestRelease: CurrentRelease = {
-  version: '24.1.0',
-  date: 'March 31, 2024',
-  mac12_arm: 'https://github.com/NREL/EnergyPlus/releases/download/v24.1.0/EnergyPlus-24.1.0-9d7789a3ac-Darwin-macOS12.1-arm64.dmg',
-  mac12: 'https://github.com/NREL/EnergyPlus/releases/download/v24.1.0/EnergyPlus-24.1.0-9d7789a3ac-Darwin-macOS12.1-x86_64.dmg',
-  mac11: 'https://github.com/NREL/EnergyPlus/releases/download/v24.1.0/EnergyPlus-24.1.0-9d7789a3ac-Darwin-macOS11.6-x86_64.dmg',
-  windows_main: 'https://github.com/NREL/EnergyPlus/releases/download/v24.1.0/EnergyPlus-24.1.0-9d7789a3ac-Windows-x86_64.exe',
-  ubuntu_22: 'https://github.com/NREL/EnergyPlus/releases/download/v24.1.0/EnergyPlus-24.1.0-9d7789a3ac-Linux-Ubuntu22.04-x86_64.run',
-  ubuntu_20: 'https://github.com/NREL/EnergyPlus/releases/download/v24.1.0/EnergyPlus-24.1.0-9d7789a3ac-Linux-Ubuntu20.04-x86_64.run'
+  version: '24.2.0',
+  date: 'September 30, 2024',
+  mac13_arm: 'https://github.com/NREL/EnergyPlus/releases/download/v24.2.0a/EnergyPlus-24.1.0-94a887817b-Darwin-macOS13-arm64.dmg',
+  mac12: 'https://github.com/NREL/EnergyPlus/releases/download/v24.2.0a/EnergyPlus-24.1.0-94a887817b-Darwin-macOS12.1-x86_64.dmg',
+  windows_main: 'https://github.com/NREL/EnergyPlus/releases/download/v24.2.0a/EnergyPlus-24.1.0-94a887817b-Windows-x86_64.exe',
+  ubuntu_22: 'https://github.com/NREL/EnergyPlus/releases/download/v24.2.0a/EnergyPlus-24.1.0-94a887817b-Linux-Ubuntu22.04-x86_64.run',
+  ubuntu_24: 'https://github.com/NREL/EnergyPlus/releases/download/v24.2.0a/EnergyPlus-24.1.0-94a887817b-Linux-Ubuntu24.04-x86_64.run'
```

Comment:
These binaries aren't all posted yet, so the links won't work as I type this, but they should be done within an hour or so.